curp/src/members.rs

```diff
@@ -98,9 +98,16 @@ impl ClusterInfo {
     }
 
     /// Calculate the member id
-    fn calculate_member_id(address: &str, cluster_name: &str, timestamp: Option<u64>) -> ServerId {
+    fn calculate_member_id(
+        mut addrs: Vec<String>,
+        cluster_name: &str,
+        timestamp: Option<u64>,
+    ) -> ServerId {
         let mut hasher = DefaultHasher::new();
-        hasher.write(address.as_bytes());
+        addrs.sort();
```

Comment:
Why should we sort addrs?

Reply:
Mentioned in https://github.com/xline-kv/Xline/pull/423#discussion_r1323160438